google-home-community.groovy

```diff
@@ -2628,6 +2682,9 @@ private executeCommand_ThermostatTemperatureSetpoint(deviceInfo, command) {
         setpoint = celsiusToFahrenheitRounded(setpoint)
     }
 
+    // round to 0.1 degree resolution
+    setpoint = (setpoint.toFloat()*10).toInteger()/10
```

Comment:
This seems unrelated.  Also `setpoint` should already be rounded to one decimal place.

Reply:
Not sure if its because I'm using C instead of F?  The set points coming back from GHC to Hubitat in this case have 10 decimal place precision.

I can take this out if you like (or different PR)?

Reply:
I see.  Are you setting the thermostat in F in Google Home, but using C in Hubitat?  I see Google does the conversion out to many decimal places.  I'll make a separate PR to clean that up.